pre-commit

```diff
@@ -130,6 +131,19 @@ if [ ${#STAGED_PHP_FILES[@]} != 0 ]; then
 	fi
 fi
 
+# Run Codeception tests
+if [ "$CODECEPTION_CHECK" == 1 ]; then
+	if [ -f "$SELF_DIR/codeception.yml" ]; then
+		# Download if it does not exist
+		if [ ! -f "./dev-lib/codeception.phar" ];  then
+			wget -O "./dev-lib/codecept.phar" http://codeception.com/codecept.phar
+		fi
+		php dev-lib/codecept.phar run
```

Comment:
@shadyvb one thing: should we instead just store `codecept.phar` in `/tmp`? If the user doesn't add `dev-lib` as a submodule with this specific name, then the Codeception test would fail since the path wouldn't exist.